midenc-tx-kernel/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::Arc;
+
+use miden_assembly::{
+    ast::ModuleKind, library::CompiledLibrary, Assembler, Compile, CompileOptions,
+    DefaultSourceManager,
+};
+
+/// Stubs for the Miden rollup tx kernel
+pub struct MidenTxKernelLibrary(CompiledLibrary);
+
+impl AsRef<CompiledLibrary> for MidenTxKernelLibrary {
+    fn as_ref(&self) -> &CompiledLibrary {
+        &self.0
+    }
+}
+
+impl From<MidenTxKernelLibrary> for CompiledLibrary {
+    fn from(lib: MidenTxKernelLibrary) -> Self {
+        lib.0
+    }
+}
+
+impl Default for MidenTxKernelLibrary {
+    fn default() -> Self {
+        // TODO: Load compiled MASL from file
+        let source_manager = Arc::new(DefaultSourceManager::default());
+        let assembler = Assembler::new(source_manager.clone());
+        let tx_module = (include_str!("../masm/tx.masm"))
```

Comment:
```suggestion
        let tx_source = source_manager.load("midenc-tx-kernel/masm/tx.masm", include_str!("../masm/tx.masm"));
        let tx_module = tx_source
```

This ensures that the debug info for these files can be found if the `midenc-tx-kernel` directory is available when running tests or when executing programs containing its code.

Reply:
Thanks! Done in https://github.com/0xPolygonMiden/compiler/pull/277/commits/ab2b7257965b0f6e56d75e84f403d3b9fd683c4b